projects/adt7420-pmdz/src/examples/dummy/dummy_example.c

```diff
@@ -85,6 +85,8 @@ int dummy_example_main()
 	if (ret)
 		goto error_adt7420;
 
+	no_os_mdelay(240);
```

Comment:
unrelated change, which I think is already in another PR

Reply:
I've specified that the other PR should be merged first.

Reply:
Yep, saw that only after leaving this comment.